src/main/docs/guide/httpServer/customArgumentBinding.adoc

```diff
@@ -0,0 +1,59 @@
+
+== Custom argument binding
+
+Micronaut uses an api:http.bind.binders.ArgumentBinderRegistry[ArgumentBinderRegistry] to look up api:core.bind.ArgumentBinder[ArgumentBinder]s
+capable of binding to the arguments in the controller methods. The default implementation will look for an annotation on
+the argument that is, itself, annotated with `@Bindable`. If one exists the argument binder registry will search for an
+argument binder that supports that annotation.
+
+If no fitting annotation is found Micronaut will try to find an argument binder that supports the argument's type.
+
+An api:core.bind.ArgumentBinder[ArgumentBinder] can be registered as a bean. In this case it is only present when a
+api:context.BeanContext[BeanContext] is initialized.
+
+Binders are called by the same server logic that calls the api:io.micronaut.http.annotation.Controller[Controller]. Once
```

Comment:
This first sentence doesn't really add value. It's not relevant what calls the logic as that isn't what is being documented